maintenance/in_place_import.sh

```diff
@@ -29,9 +33,10 @@ HOST=${HOST:-outreach.openmicroscopy.org}
 FOLDER=${FOLDER:-siRNAi-HeLa}
 NUMBER=${NUMBER:-40}
 USER=${USER:-user}
-for i in {1..$NUMBER}
+for ((i=1;i<=$NUMBER;i++));
```

Comment:
not sure this helps at all?

Reply:
it does. We can now run the script without issue either with ``bash`` or ``sh``

Reply:
That's interesting:
```console
mtbc@ls28101:/tmp$ cat test.sh 
NUMBER=4
for ((i=1;i<=$NUMBER;i++));
do  echo $i
done
mtbc@ls28101:/tmp$ sh test.sh 
test.sh: 2: test.sh: Syntax error: Bad for loop variable
mtbc@ls28101:/tmp$ bash test.sh 
1
2
3
4
```

Reply:
@mtbc yes, but the script works, both with ``bash`` as well as with ``sh``. Don't know why.
But that is good.